woocommerce/woocommerce
# Add Kliken to default OBW extension list

### Changes proposed in this Pull Request:

Closes #52082

Adds Kliken Meta Ads & Pixel to Core Profiler default free extensions list that replaces Pinterest 50% of the time



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com`
2. Disable marketplace suggestion and save
1. From WC core, run `wp option set woocommerce_remote_variant_assignment 59` 
1. From WC core, go to Core profiler extensions step
1. See that Pinterest is offered
1. From WC core, run `wp option set woocommerce_remote_variant_assignment 61` 
3. Refresh core profiler extensions page
1. See that `Meta Ads & Pixel for WooCommerce` is offered in place of Pinterest extension(https://github.com/woocommerce/woocommerce/issues/52082)
2. De-select other extensions and select `Grow your business with Facebook and Instagram` and click next
4. Go to `Plugins > Installed plugins`
5. Observe `Kliken: Ads + Pixel for Meta` is installed and activated

<img width="1132" alt="image" src="https://github.com/user-attachments/assets/16618c48-7a97-4a46-93e9-709cecc712f1">



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
